### ISSUE TITLE
🐛 Printing special terminal control characters "^[]11;rgb:0000/0000/0000^G"

### ISSUE THREAD

**Issue by yang** [dandavison/delta#1707]
.gitconfig has

```
[interactive]
  diffFilter = delta --color-only
```

Running normally is fine.

But once in GNU screen, `git diff` results in extra chars being printed before and after the prompt:

```
^[]11;rgb:0000/0000/0000^G ... 11;rgb:0000/0000/0000
```

![image](https://github.com/dandavison/delta/assets/7129/2726e189-f0c9-47cf-9fc8-1319df71a8bf)

(Similar strangeness happening with using as pager.)

**Comment by bash** [comment]
Hmm looks like something is going wrong with the dark/light mode detection introduced in #1615.

As a workaround you can disable this by passing `--dark` or `--light` to delta:

```gitconfig
[interactive]
	diffFilter = delta --color-only --dark
```

This is likely a bug in either [`terminal-colorsaurus`](https://github.com/bash/terminal-colorsaurus) (the library delta uses to detect the terminal's color scheme) or screen.

**Comment by yang** [comment]
That did it, thank you!

**Comment by bash** [comment]
This should be fixed now that 0.18.0 has been released.

Automatic color detection is now disabled in screen, so the `--dark` / `--light` workaround isn't needed anymore.

**Comment by th1000s** [comment]
Nice, thanks @bash!
